test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,8 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+  it("returns Array of human, cat and dog years when passed human years", () => {
+    expect(humanCatDogYears(12)).toEqual([12, 64, 74]);
+    expect(humanCatDogYears(17)).toEqual([17, 84, 99]);
+  });
+});
```

Comment:
you should add more tests for edge cases, e.g. when human years is 0, 1 or 2. e.g.
```
it("returns correct years when human years is equal to 0", () => {
  expect(humanCatDogYears(0)).toEqual([0, 0 ,0]);
}

it("returns correct years when human years is equal to 1", () => {
  expect(humanCatDogYears(1)).toEqual([1, 15, 15]);
}

it("returns correct years when human years is equal to 2", () => {
  expect(humanCatDogYears(1)).toEqual([2, 24, 24]);
}
```